packages/hardhat-core/sample-projects/javascript-esm/test/Lock.js

```diff
@@ -3,7 +3,9 @@ import {
   loadFixture,
 } from "@nomicfoundation/hardhat-toolbox/network-helpers.js";
 import { anyValue } from "@nomicfoundation/hardhat-chai-matchers/withArgs.js";
-import { expect } from "chai";
+import chai from "chai";
+
+const { expect } = chai;
```

Comment:
This was causing a "no named export" kind of error in the CI when windows and yarn were being used. I have no idea why it only happened in that scenario (and we couldn't reproduce it locally), but this fixed it. And since this version of chai is CJS-only, it seems like a reasonable thing to do anyway.

Reply:
That is weird, because this code would transpile to cjs anyway. I have no problem with it, I was just curious.